test/DaedalusCompiler.Tests/CompilerFullTest.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Linq;
+using DaedalusCompiler.Compilation;
+using DaedalusCompiler.Dat;
+using Xunit;
+
+namespace DaedalusCompiler.Tests
+{
+    public class CompilerFullTest
+    {
+        private string _code;
+        private AssemblyBuilder _assemblyBuilder;
+
+        public CompilerFullTest()
+        {
+            _assemblyBuilder = new AssemblyBuilder();
+        }
+
+        private DatFile compileCodeAndGetDATFileOfThat()
+        {
+            Utils.WalkSourceCode(_code, _assemblyBuilder);
+            var datBuilder = new DatBuilder(_assemblyBuilder);
+            var datFile = datBuilder.GetDatFile();
+            var datToReturn = new DatFile();
+            
+            datToReturn.Load(datFile.getBinary());
+
+            return datToReturn;
+        }
+        
+        [Fact]
+        public void TestIfSymbolsAreInDatFile()
+        {
+            _code = @"
+                var int x;
+                var int y;
+                var int z;
+
+                func void testFunc() {
+                };
+            ";
+
+            var datFile = compileCodeAndGetDATFileOfThat();
+            
+            Assert.Equal(datFile.Symbols.Count(), 4);
```

Comment:
this check is ok but wouldn't it be better if we first compile the code in Gothic (by hand), then use our DatLoader to load the compiled datFile and then compare our and Gothic's file. (symbols' count, instrunctions, etc.)